palladian-commons/pom.xml

```diff
@@ -44,5 +44,27 @@
 			<artifactId>commons-codec</artifactId>
 			<version>1.9</version>
 		</dependency>
+		<dependency>
+		  <groupId>org.powermock</groupId>
+		  <artifactId>powermock-api-mockito</artifactId>
+		  <version>1.6.5</version>
+		</dependency>
+		<dependency>
+			  <groupId>org.powermock</groupId>
```

Comment:
Fix the indenting here and in the rest of the POM file, to match with the existing structure.